scripts/fetch-test-binaries.sh

```diff
@@ -1 +1,61 @@
 #!/usr/bin/env bash
```

Comment:
some providers just tell you where to get this script. Not sure whether it's better to "freeze" it here, or point elsewhere so any changes don't need to be pulled...